### DISCUSSION TITLE
Set number of boxes columns as a setting

### DISCUSSION THREAD

**Discussion by PubliAlex** [OFFLINE-GmbH/oc-boxes-support#7] [Ideas and feature requests]
Hello,

For the boxes I create for my customers, I display a screenshot of the box result as an illustration image.

I can do it by defining the path to my .jpg in the icon parameter of the box configuration, the problem is that thumbs in the boxes selection screen are way to small. See an example : 

![image](https://user-images.githubusercontent.com/55833027/188445363-a1a69686-6b86-44a0-80f0-1799a32b9662.png)

By default, it display 5 boxes per lines, it could be useful to be able to change that value to have bigger boxes.

Best regards,

Alex

**Comment by tobias-kuendig** [comment]
Thank you Alex, this is now configurable in version v2.0.58. You can set a default value in the backend settings. You can also change the number when selecting a partial.

Furthermore, the `.partial-icon` and `.oc-boxes-partial-selection-modal` classes have been added to provide easier custom styling.
